common/lib/xmodule/xmodule/sgp_block.py

```diff
@@ -0,0 +1,205 @@
+"""
+XBlock for Staff Graded Problem
+"""
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+import csv
+import hashlib
+import io
+import json
+import logging
+
+import markdown
+from crum import get_current_request
+from django.middleware.csrf import get_token
+from web_fragments.fragment import Fragment
+from webob import Response
+from xblock.core import XBlock
+from xblock.fields import Float, Scope, String
+from xblock.runtime import NoSuchServiceError
+from xblock.scorable import ScorableXBlockMixin, Score
+from xblockutils.resources import ResourceLoader
+from xblockutils.studio_editable import StudioEditableXBlockMixin
+
+
+_ = lambda text: text
```

Comment:
This gave me pause so I found other places where this was done and they referenced https://openedx.atlassian.net/browse/PLAT-902, which makes me think this problem may no longer be an issue since the earliest django version we use is 1.11 at this point, no?